test/libcommonserver/io/testcheckifpathexists.cpp

```diff
@@ -454,6 +454,17 @@ void TestIo::testCheckIfPathExistsWithSameNodeIdAllBranches() {
 
         _testObj->resetFunctions();
     }
+
+    // Checking existence of a subdirectory inside a directory that have been deleted and replaced with a file with the same name
+    // ex: conversion of a bundle into a single file (macOS)
+    {
+        const SyncPath path = _localTestDirPath / "test_pictures/picture-1.jpg/A";
+        bool exists = false;
+        IoError ioError = IoErrorUnknown;
+        CPPUNIT_ASSERT(_testObj->checkIfPathExists(path, exists, ioError));
+        CPPUNIT_ASSERT(!exists);
+        CPPUNIT_ASSERT(ioError == IoErrorNoSuchFileOrDirectory);
```

Comment:
```suggestion
        CPPUNIT_ASSERT_EQUAL(IoErrorNoSuchFileOrDirectory, ioError);
```
Out of ignorance, I haven't  used the suggested syntax for comparing two values (Hervé right lyuses it). I think it displays a more precise message in case of failure.

Reply:
Indeed, that's better. Thx